_pytest/assertion/__init__.py

```diff
@@ -113,7 +115,8 @@ def callbinrepr(op, left, right):
         for new_expl in hook_result:
             if new_expl:
                 if (sum(len(p) for p in new_expl[1:]) > 80*8
-                        and item.config.option.verbose < 2):
+                        and item.config.option.verbose < 2
+                        and 'CI' not in os.environ):
```

Comment:
Is this meant to support most systems out there, or is the user supposed to set that? I ask because I don't think Jenkins [sets that variable](https://wiki.jenkins-ci.org/display/JENKINS/Building+a+software+project).

(I ask because if your intention is for users to set it, we should mention it in the docs otherwise users would expect this to work out of the box).